src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1663,9 +1677,25 @@ namespace Sched {
                     }
                 }
 
+                // curcolCount is 1-indexed here
+                if (static_cast<size_t>(curcolCount) > result->second["values"].size()) {
+                    ShowSevereCustom(
+                        state, eoh, format("Requested column number {}, but found only {} columns.", curcolCount, result->second["values"].size()));
+                    ShowContinueError(state, fmt::format("Error Occurred in {}", state.files.TempFullFilePath.filePath));
+                    ShowFatalError(state, "Program terminates due to previous condition.");
+                }
                 auto const &column_json = result->second["values"][curcolCount - 1];
                 rowCnt = column_json.size();
-                auto const column_values = column_json.get<std::vector<Real64>>(); // (AUTO_OK_OBJ)
+
+                std::vector<Real64> column_values;
+                try {
+                    column_values = column_json.get<std::vector<Real64>>();
+                } catch (nlohmann::json::type_error &e) {
+                    ShowSevereCustom(state, eoh, format("Column number {} has non-numeric data.", curcolCount));
+                    ShowContinueError(state, e.what());
+                    ShowContinueError(state, fmt::format("Error Occurred in {}", state.files.TempFullFilePath.filePath));
+                    ShowFatalError(state, "Program terminates due to previous condition.");
+                }
```

Comment:
Gracefully handle using a column that had a null (or a string for that matter) instead of throwing a crytpic json error